tests/cypress/e2e/textFieldInitializerTest.cy.ts

```diff
@@ -0,0 +1,455 @@
+import {createSite, deleteSite} from '@jahia/cypress';
+import {PageComposer} from '../page-object/pageComposer';
+
+describe('Test the text field initializer)', () => {
+    const siteKey = 'extFieldInitializerTest';
+    const langEN = 'en';
+    const langFR = 'fr';
+    const langDE = 'de';
+    const languages = langEN + ',' + langFR + ',' + langDE;
+    const siteConfig = {
+        languages: languages,
+        templateSet: 'dx-base-demo-templates',
+        serverName: 'localhost',
+        locale: langEN
+    };
+
+    before(function () {
+        createSite(siteKey, siteConfig);
+        cy.runProvisioningScript({fileName: 'provisioning/enableModule.yaml', type: 'application/yaml'});
+    });
+
+    after(function () {
+        deleteSite(siteKey);
+    });
+
+    const checkFieldValues = (contentEditor, fields, lang) => {
+        // eslint-disable-next-line guard-for-in
+        for (const fieldId in fields) {
+            const field = fields[fieldId];
+            switch (field.type) {
+                case 'smallText':
+                    contentEditor.getSmallTextField(fieldId).checkValue(field.values[lang]);
+                    break;
+                case 'date':
+                    contentEditor.getDateField(fieldId).checkValue(field.values[lang]);
+                    break;
+                default:
+                    cy.log('unknown type field');
+            }
+        }
+    };
+
+    const checkValuesDisplayedInPageComposer = (pageComposer, valuesToCheck, lang) => {
+        const values = valuesToCheck[lang];
+        values.forEach(value => {
+            pageComposer.shouldContain(value);
+        });
+    };
+
+    const editFieldValues = (contentEditor, fields, lang) => {
+        // eslint-disable-next-line guard-for-in
+        for (const fieldId in fields) {
+            const field = fields[fieldId];
+            switch (field.type) {
+                case 'smallText':
+                    contentEditor.getSmallTextField(fieldId).addNewValue(field.values[lang], true);
```

Comment:
Be careful with the `force: true`  it will prevent the scroll but maybe in some case you will need to scroll to edit a field

Reply:
ok, done